ClassesAndObjects/Classes4/Movie.cs

```diff
@@ -46,7 +46,7 @@ public void SetTitle(string aTitle)
             title = aTitle;
         }
 
-        public bool quals(object obj)
+        public bool ChekingMovies(object obj)
```

Comment:
ChekingMovies -- šis apzīmē darbību, bet tev vajag jautājumu, lai abildētu ar true/false

Reply:
IsMovieThere